api/dps/server.go

```diff
@@ -50,7 +50,7 @@ func NewServer(index index.Reader) *Server {
 	return &s
 }
 
-// GetFirst implements the `GetFirst` function of the generated GRPC server.
+// GetFirst implements the `GetFirst` method of the generated GRPC server.
```

Comment:
If you want to split hairs, Rob Pike was always insistent that there are no "methods" in Go, only functions with receivers. I know even the core Go docs aren't consistent on that, but there you go anyway :P

Reply:
While I agree with his point technically, but I like calling them methods because then it makes it easier to know what we're talking about 😄 In this case it doesn't matter much though :p